aztec/src/main/kotlin/org/wordpress/aztec/toolbar/AztecToolbar.kt

```diff
@@ -561,22 +572,37 @@ class AztecToolbar : FrameLayout, IAztecToolbar, OnMenuItemClickListener {
         }
     }
 
+    private fun syncEditorFromSource() {
```

Comment:
It seems to me that this method reads the content from the Visual editor, and in case of changes put the content into the HTML editor. 
If so, should we rename this method in `syncSourceEditorFromVisual`?

Reply:
Indeed, binary is hard! Will swap the names, good eyes!